pkg/registry/apis/dashboard/legacy/sql_dashboards.go

```diff
@@ -410,6 +412,11 @@ func (a *dashboardSqlAccess) buildSaveDashboardCommand(ctx context.Context, orgI
 		}
 	}
 
+	// v1 should be saved as schema version 41. v0 allows for older versions
+	if strings.HasSuffix(dash.APIVersion, "v1alpha1") && schemaversion.GetSchemaVersion(dash.Spec.Object) < int(schemaversion.LATEST_VERSION) {
+		dash.APIVersion = dashboardv0.VERSION
```

Comment:
🤔 is the APIVersion "v0" or "dashboard.grafana.app/v0"

Reply:
it's `dashboard.grafana.app/v0alpha1`

Reply:
See here: https://github.com/grafana/grafana/blob/main/pkg/registry/apis/dashboard/legacy/sql_dashboards.go#L286

Reply:
```suggestion
		dash.APIVersion = dashboardv0.APIVERSION
```
VERSION is just "v0alpha1" we need the group also

Reply:
ha... ok does not really matter because on line 410 (three below) we trim the prefix

Reply:
I follow the definition like 10 times and always looked in the wrong line and looked at `APIVERSION`, thanks for spotting it!

Yeah, not effect, but i'll make it right